wiki/Beat/id.md

```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - ketukan
+  - garis putih
+  - garis-garis putih
+  - ketukan putih
+  - ketukan-ketukan putih
+  - keseluruhan ketukan
```

Comment:
`whole beat` -- given its musical term -- doesn't literally translate into `keseluruhan ketukan` tho...

it's more of a `satu ketukan` or `satu ketukan penuh` instead

Reply:
is any better or common usage while we can adds more

Reply:
i don't think there's any other alternative (that i'm aware of, at least) tho...

however i'm *very* sure that the term `whole beat` doesn't translate into `keseluruhan ketukan`, yeah

Reply:
i'd fixed this